### PR TITLE
Reintroduce unit testing demo and exercises

### PR DESCRIPTION
**Pull request by onemanwenttomow**

### CODE REVIEW

File: sessions/js-unit-testing/demo-end/.codesandbox/tasks.json
Comment (NikRoe) [inline sessions/js-unit-testing/demo-end/.codesandbox/tasks.json:1]: we should be able to remove this file safely, right?

File: sessions/js-unit-testing/demo-end/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Use `npm run test` to run the tests.
+
+### CodeSandbox Cloud
+
+Select the "Preview: 3000" tab to view this project.
+
+Select the "Tests: logs" tab to view the tests.
+
+> The `npm run start` and `npm run test` scripts run automatically.
```
Comment (NikRoe) [inline sessions/js-unit-testing/demo-end/README.md:24]:
```suggestion
```

File: sessions/js-unit-testing/demo-start/.codesandbox/tasks.json
Comment (NikRoe) [inline sessions/js-unit-testing/demo-start/.codesandbox/tasks.json:1]: this file should be removable as well

File: sessions/js-unit-testing/demo-start/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Use `npm run test` to run the tests.
+
+### CodeSandbox Cloud
+
+Select the "Preview: 3000" tab to view this project.
+
+Select the "Tests: logs" tab to view the tests.
+
+> The `npm run start` and `npm run test` scripts run automatically.
```
Comment (NikRoe) [inline sessions/js-unit-testing/demo-start/README.md:30]:
```suggestion
```

File: sessions/js-unit-testing/demo-start/sandbox.config.json
Comment (NikRoe) [inline sessions/js-unit-testing/demo-start/sandbox.config.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/demo-end/sandbox.config.json
Comment (NikRoe) [inline sessions/js-unit-testing/demo-end/sandbox.config.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/get-animal/.codesandbox/tasks.json
Comment (NikRoe) [inline sessions/js-unit-testing/get-animal/.codesandbox/tasks.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/get-animal/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Use `npm run test` to run the tests.
+
+### CodeSandbox Cloud
+
+Select the "Preview: 3000" tab to view this project.
+
+Select the "Tests: logs" tab to view the tests.
+
+> The `npm run start` and `npm run test` scripts run automatically.
```
Comment (NikRoe) [inline sessions/js-unit-testing/get-animal/README.md:38]:
```suggestion
```

File: sessions/js-unit-testing/get-animal/sandbox.config.json
Comment (NikRoe) [inline sessions/js-unit-testing/get-animal/sandbox.config.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/math-functions/.codesandbox/tasks.json
Comment (NikRoe) [inline sessions/js-unit-testing/math-functions/.codesandbox/tasks.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/math-functions/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Use `npm run test` to run the tests.
+
+### CodeSandbox Cloud
+
+Select the "Preview: 3000" tab to view this project.
+
+Select the "Tests: logs" tab to view the tests.
+
+> The `npm run start` and `npm run test` scripts run automatically.
```
Comment (NikRoe) [inline sessions/js-unit-testing/math-functions/README.md:62]:
```suggestion
```

File: sessions/js-unit-testing/math-functions/sandbox.config.json
Comment (NikRoe) [inline sessions/js-unit-testing/math-functions/sandbox.config.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/people/.codesandbox/tasks.json
Comment (NikRoe) [inline sessions/js-unit-testing/people/.codesandbox/tasks.json:1]: we should be able to remove this safely

File: sessions/js-unit-testing/people/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Use `npm run test` to run the tests.
+
+### CodeSandbox Cloud
+
+Select the "Preview: 3000" tab to view this project.
+
+Select the "Tests: logs" tab to view the tests.
+
+> The `npm run start` and `npm run test` scripts run automatically.
```
Comment (NikRoe) [inline sessions/js-unit-testing/people/README.md:46]:
```suggestion
```

File: sessions/js-unit-testing/people/sandbox.config.json
Comment (NikRoe) [inline sessions/js-unit-testing/people/sandbox.config.json:1]: we should be able to remove this safely

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Pete,

thanks for the work you put into this!

I think the guide and exercise are looking fine.
However, I noticed that there are some files for codesandbox and some mentions in the README files on how to run the project in the codesandbox cloud. We should remove those files and mentions.

Thanks in advance 🙏

**Comment by onemanwenttomow** [comment]
> Hi Pete,
> 
> thanks for the work you put into this!
> 
> I think the most is looking fine. However, I noticed that there are some files for codesandbox and some mentions in the README files on how to run the project in the codesandbox cloud. We should remove those files and mentions.
> 
> Thanks in advance 🙏

Good points @NikRoe ! Thanks for the review

**Review by NikRoe — APPROVED** [review]
🥳

**Pull request merged**